pulsar-broker/src/main/java/org/apache/pulsar/broker/admin/impl/PersistentTopicsBase.java

```diff
@@ -322,20 +322,23 @@ protected void internalGrantPermissionsOnTopic(final AsyncResponse asyncResponse
                 });
     }
 
-    protected void internalDeleteTopicForcefully(boolean authoritative) {
-        validateTopicOwnership(topicName, authoritative);
-        validateNamespaceOperation(topicName.getNamespaceObject(), NamespaceOperation.DELETE_TOPIC);
+    protected void internalDeleteTopicForcefully(AsyncResponse asyncResponse, boolean authoritative) {
```

Comment:
`AsyncResponse` is not allowed to pass here. Please ref https://github.com/apache/pulsar/issues/14365